app/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnector.scala

```diff
@@ -47,17 +47,21 @@ class EligibilityCheckConnectorImpl @Inject() (http:              WSHttp,
 
   val itmpBaseURL: String = baseUrl("itmp-eligibility-check")
 
-  def url(nino: String): String =
-    s"$itmpBaseURL/help-to-save/eligibility-check/$nino"
+  def url(nino: String, ucResponse: Option[UCResponse]): String = {
+    ucResponse match {
+      case Some(UCResponse(a, b)) ⇒ s"$itmpBaseURL/help-to-save/eligibility-check/$nino?ucClaimant=$a&withinThreshold=${b.getOrElse(false)}"
```

Comment:
if `withinThreshold` is `None` we don't want to send that parameter to DES. We'd do something like this:
```scala
s"$itmpBaseURL/help-to-save/eligibility-check/$nino?ucClaimant=$a${b.map(x => s"&withinThreshold=$x").getOrElse("")}"
```

Reply:
we basically want to pass in the parameters as DWP give them to us where the `withinThreshold` field is optional

Reply:
ah yes, i missed the important bit -  fixing it now..